test/preview.test.js

```diff
@@ -38,20 +38,38 @@ describe('Test preview plugin', () => {
   });
 
   it('Preview plugin switches to preview from gdrive URL', async () => {
-    const { popupOpened } = await new SidekickTest({
+    const test = new SidekickTest({
       page,
       setup: 'pages',
       url: 'https://docs.google.com/document/d/2E1PNphAhTZAZrDjevM0BX7CZr7KjomuBO6xE1TUo9NU/edit',
       plugin: 'preview',
       waitPopup: 2000,
-    }).run();
+    });
+    test.apiResponses[0].edit.sourceLocation = 'gdrive:1mfBb_tpzM4yYGdxMhRgrKEnBqboxsr';
+    test.apiResponses[0].edit.contentType = 'application/vnd.google-apps.document';
+    const { popupOpened } = await test.run();
     assert.strictEqual(
       popupOpened,
       'https://main--pages--adobe.hlx.page/creativecloud/en/test',
       'Preview URL not opened',
     );
   }).timeout(IT_DEFAULT_TIMEOUT);
 
+  it('Preview plugin fails for office document from gdrive URL', async () => {
```

Comment:
Let's move this with the other edit-specific preview tests below.
```suggestion
  it('Edit-specific preview plugin fails for office document from gdrive URL', async () => {
```